File: programming/1.2/main.cpp
#include <iostream>
#include <cmath>


using namespace std;

int main() {
  int x, y;

  cout << "Enter x: ";
  cin >> x;

  cout << "Enter y != x: ";
  cin >> y;

  if (x == y) {
    cout << "You enter x == y";
    exit(1);
  }

  float c, d, z;

  c = sqrt(x + y);
  d = cos(x * y);
  z = (atan(x + y) - 1.5 * exp(x) * c + sqrt(abs(d)));

  cout << "Result = " << z;

  return 0;
}

File: programming/1.2/main.py
from math import sin, sqrt, cos, atan, exp
from random import randint


x = randint(0, 10)
y = randint(0, 10)

while x == y: 
  y = randint(0, 10)


c = sqrt(x + y)
d = cos(x * y)
z = (atan(x + y) - 1.5 * exp(x) * c + sqrt(abs(d)))

print(z)
